From 574fb2fec1dbf7d13cc91b36d5d397dbce56419d Mon Sep 17 00:00:00 2001
From: Sridharan S N <quic_sridsn@quicinc.com>
Date: Wed, 3 Jan 2024 23:49:41 +0530
Subject: [PATCH 450/500] drivers: soc: qcom: Add support for
 QCOM_SOC_NETSTANDBY

Adding the build support for netstandby feature.

Change-Id: I11ff7784dac54fd26a0e703b5825a98df2978bd5
Signed-off-by: Hariharan K <quic_harihk@quicinc.com>
Signed-off-by: Sridharan S N <quic_sridsn@quicinc.com>
---
 drivers/soc/qcom/Kconfig  | 6 ++++++
 drivers/soc/qcom/Makefile | 1 +
 2 files changed, 7 insertions(+)

diff --git a/drivers/soc/qcom/Kconfig b/drivers/soc/qcom/Kconfig
index 71b458ad3729..ce931affbb56 100644
--- a/drivers/soc/qcom/Kconfig
+++ b/drivers/soc/qcom/Kconfig
@@ -339,6 +339,12 @@ config IPQ_REG_UPDATE
 	help
 	 Generic driver to configure required registers
 
+config QCOM_SOC_NETSTANDBY
+	bool "Qualcomm Technologies, Inc. SoC network standby driver"
+	depends on ARCH_QCOM
+	help
+	  SoC network standby driver for ErP support
+
 config QTI_TZ_LOG
 	tristate "Trust Zone (TZ) Log Driver"
 	depends on DEBUG_FS
diff --git a/drivers/soc/qcom/Makefile b/drivers/soc/qcom/Makefile
index 9af1468921a0..669e623d7d28 100644
--- a/drivers/soc/qcom/Makefile
+++ b/drivers/soc/qcom/Makefile
@@ -38,6 +38,7 @@ obj-$(CONFIG_QCOM_DCC_V2)	+= dcc_v2.o
 obj-$(CONFIG_IPQ_DEBUG)	+= ipq-debug.o
 obj-$(CONFIG_MSM_RPM_LOG)	+= rpm_log.o
 obj-$(CONFIG_IPQ_REG_UPDATE) += ipq_reg_update.o
+obj-$(CONFIG_QCOM_SOC_NETSTANDBY) += netstandby.o
 obj-$(CONFIG_QTI_TZ_LOG) += tz_log.o
 obj-$(CONFIG_QTI_LICENSE_MANAGER) += license_manager.o
 obj-$(CONFIG_QTI_LICENSE_MANAGER)	+= qwes.o
-- 
2.34.1

